warehouse/admin/templates/admin/users/detail.html

```diff
@@ -988,6 +988,41 @@ <h3 class="card-title">Pending OpenID Connect Publishers</h3>
         </div>
       </div> <!-- .card -->
 
+      <div class="card">
+        <div class="card-header with-border">
+          <h3 class="card-title">Unique logins</h3>
+        </div>
+
+        <div class="card-body">
+          {% if user.unique_logins %}
+          <div class="table-responsive p-0">
+            <table class="table table-hover" id="pending-oidc-publishers">
+              <thead>
+                <tr>
+                  <th scope="col">Created</th>
+                  <th scope="col">IP address</th>
+                  <th scope="col">Status</th>
+                  <th scope="col">Device Information</th>
+                </tr>
+              </thead>
+              <tbody>
+                {% for login in user.unique_logins %}
```

Comment:
we are almost certainlty going to need to paginate this... but for now seems fine since its admin.